pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
 
 dependencies = [
     "toml >= 0.10.2",
+    "requests >= 2.32.0",
```

Comment:
Add library to list of 3rd party libs in the README.

Reply:
Ups, forget the README- will come ....

Reply:
Updated requirements*.txt as well.